Parsers/ASimAuthentication/ProductParsers/vimAuthenticationPostgreSQL copy.yaml

```diff
@@ -0,0 +1,205 @@
+Parser:
+  Title: ASIM PostgreSQL authentication
+  Version: '0.1.1'
+  LastUpdated: April 20, 2022
+Product:
+  Name: PostgreSQL
+Normalization:
+  Schema: Authentication
+  Version: '0.1.1'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+      This Query Parser maps PostgreSQL sign in logs to the Advanced Security Information Model authenticaion schema.
+ParserName: ASimAuthenticationPostgreSQL
+ParserParams:
```

Comment:
The YAML file needs to include the additional parameters as well.

Reply:
fixed